fs/xfstests.py

```diff
@@ -221,6 +229,90 @@ def setUp(self):
         self.test_mnt = self.params.get('test_mnt', default='/mnt/test')
         self.disk_mnt = self.params.get('disk_mnt', default='/mnt/loop_device')
         self.fs_to_test = self.params.get('fs', default='ext4')
+        self.run_type = self.params.get('run_type', default='distro')
+
+        if self.run_type == 'upstream':
+            prefix = "/usr/local"
+            bin_prefix = "/usr/local/bin"
+            sbin_prefix = "/usr/local/bin"
+
+            if self.detected_distro.name == 'SuSE':
+                # SuSE has /sbin at a higher priority than /usr/local/bin
+                # in $PATH, so install all the binaries in /sbin to make
+                # sure they are picked up correctly by xfstests.
+                #
+                # We still install in /usr/local but binaries are kept in
+                # /sbin
+                bin_prefix = "/sbin"
+                sbin_prefix = "/sbin"
```

Comment:
Same comment. Did you actually mean /bin and /sbin ? If not then just one variable would suffice.